packages/core/src/spec/lib/utilSpec.ts

```diff
@@ -285,4 +285,16 @@ describe('util', () => {
           .toEqual('ActivityInfo.SCREEN_ORIENTATION_UNSPECIFIED');
     });
   });
+
+  describe('#escapeGradleString', () => {
+    it('Escapes single quotes', () => {
+      expect(util.escapeGradleString('Single quote \''))
+          .toEqual('Single quote \\\\\\\'');
+    });
+
+    it('Escapes forward slashes', () => {
```

Comment:
That's a backwards slash, not a forwards slash.

Reply:
done!